Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/MotionBlurTileMaxHor.azsl

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <scenesrg_all.srgi>
+#include <Atom/Features/PostProcessing/PostProcessUtil.azsli>
+#include "MotionBlur.azsli"
+
+ShaderResourceGroup PassSrg : SRG_PerPass
+{
+    Texture2D<float2> m_inputVelocity;
+    RWTexture2D<float2> m_outputVelocity;
+}
+
+[numthreads(8, 8, 1)]
+void MainCS(
```

Comment:
Add a quick comment about what the shader is doing at a high level. Please go ahead and do the same for other shaders in the PR